actions/ql/src/Security/CWE-829/UntrustedCheckoutCritical.ql

```diff
@@ -1,5 +1,5 @@
 /**
- * @name Checkout of untrusted code in trusted context
+ * @name Checkout of untrusted code in a priviledged context
```

Comment:
Typo: Should be `privileged`.
(I'm half surprised that our QL-for-QL typo check didn't catch this.)